jestGlobalSetup.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable import/no-extraneous-dependencies */
-const aggregateTranslations = require('terra-aggregate-translations');
+const aggregateTranslations = require('@cerner/terra-aggregate-translations');
```

Comment:
hmm i don't see this dep defined in our package.json, am i missing something?

Reply:
We're getting it transitively, we should add it to the package.json

Reply:
Added here https://github.com/cerner/terra-application/pull/129/commits/dc447b795056de76542c01bf94d0e810e1d9814c